src/components/contact/assets/contact.scss

```diff
@@ -0,0 +1,49 @@
+@import '~theme/base/variables';
+
+section.App__Contact {
+  display: flex;
+  margin: 40px;
+  
+  
+  .App__LeftSection {
+    flex: 0 0 40%;
+    img {
+      width: 100%;
+    }
+  }
+
+  .App__RightSection {
+    display: flex;
+    flex-flow: column;
+    margin-left: auto;
+    h1 {
+      width: 50%;
+      margin-left: auto;
+      text-align: right;
+      font-size: $fz-giant;
+      color: $color-text-heading-green;
+      text-transform: uppercase;
+      border-bottom: 1px solid $color-light-grey;
+      font-family: 'Fjalla One', sans-serif;
+      margin-bottom: 0;
+    }
+    p {
+      text-align: right;
+      width: 60%;
+      margin-left: auto;
+      color: $color-text-body-black;
+
+      .email {
+        text-decoration: none;
+        color: $color-light-green
+      }
+
+      .slackOrg {
+        color: inherit;
+        text-decoration: none;
+        border-bottom: 1px solid $color-underline-text;
+      }
+    }
+    
+  }
+}
```

Comment:
where ever you see this, can you add a new line..

Reply:
@nitish24p I tried my best to add a trailing blank line at the end of the file. DO let me know if you find any place I missed.